coalib/settings/ConfigurationGathering.py

```diff
@@ -179,13 +182,19 @@ def warn_nonexistent_targets(targets, sections, log_printer=None):
 
     # Can't be summarized as python will evaluate conditions lazily, those
     # functions have intended side effects though.
-    files_config_absent = warn_config_absent(sections, 'files')
-    bears_config_absent = warn_config_absent(sections, ['bears', 'aspects'])
-    if files_config_absent or bears_config_absent:
+    files_config_absent, arg_files = warn_config_absent(sections,
+                                                        'files',
+                                                        only_global_bears)
+    bears_config_absent, arg_bears = warn_config_absent(sections,
+                                                        ['bears', 'aspects'],
+                                                        only_global_bears)
+    if (files_config_absent or bears_config_absent) and (
+        not (only_global_bears and files_config_absent
+             and arg_files == ['files'])):
```

Comment:
@ishanSrt if `files_config_absent` is `True` then this condition will be always `false`

Reply:
I don't think thats gonna happen. if other conditions like `only_global_bears` and `arg_files == ['files']` turn false

Reply:
oh yeah right :)